src/nile/core/compile.py

```diff
@@ -67,10 +67,10 @@ def _compile_contract(
     """
 
     if account_contract or filename.endswith("Account"):
-        cmd = cmd + "--account_contract"
+        cmd = cmd + "\n--account_contract"
```

Comment:
```suggestion
        cmd.append("--account_contract")
```
isn't this better? i'm basing it on the tests

Reply:
It would be, but `cmd` is written as a string (which makes sense since they'd all be f-strings anyway):

```
    cmd = f"""
    starknet-compile {path} \
        --cairo_path={contracts_directory}
        --output {BUILD_DIRECTORY}/{filename}.json \
        --abi {ABIS_DIRECTORY}/{filename}.json
    """
```

Reply:
right!